src/components/CurrentDirectors/index.ts

```diff
@@ -0,0 +1 @@
+export { default as CurrentDirectors } from './CurrentDirectors.vue'
```

Comment:
I think, if we go down this route (folders for individual components), that we will end up with a lot of folders. There are least 2 options here:
- generalize this folder name so we can reuse it later (eg, "Directors")
- put this component in an existing, related folder (ie, "PeopleAndRoles")

Thoughts?

Reply:
I'll add it to people and roles, when I started work I thought it was gonna be more involved and use multiple files like people and roles so I duplicated it. But with how simplified it is it really doesn't benefit

Reply:
Perfect.